src/media-segment-request.js

```diff
@@ -79,13 +79,12 @@ const getProgressStats = (progressEvent) => {
     roundTripTime: roundTripTime || 0
   };
 
-  if (progressEvent.lengthComputable) {
-    stats.bytesReceived = progressEvent.loaded;
-    // This can result in Infinity if stats.roundTripTime is 0 but that is ok
-    // because we should only use bandwidth stats on progress to determine when
-    // abort a request early due to insufficient bandwidth
-    stats.bandwidth = Math.floor((stats.bytesReceived / stats.roundTripTime) * 8 * 1000);
-  }
+  stats.bytesReceived = progressEvent.loaded;
```

Comment:
If you do a math operation with `undefined` you'll get nan, so `stats.bandwidth = Math.floor((stats.bytesReceived / stats.roundTripTime) * 8 * 1000);` will be `NaN` if `bytesReceived` is `NaN`. Even if `NaN` wasn't guaranteed, compare operations with `undefined` will be false as well. I'm skeptical of setting it to any other numerical number because then we may act upon that value when really we only want to act when we know we have actual info